content/projects/python-specific/mongo/_index.md

```diff
@@ -9,7 +9,13 @@ prerequisites:
   soft: []
 ready: true
 submission_type: repo
-tags: []
+tags: 
+- docker
+- mongodb
+- postgresql
+- .yaml
+- tdd
+- unit-testing
```

Comment:
```suggestion
- tdd
- unit-testing
```
I think only one of these should be here. The instructions say 
> You can use whatever testing framework you want.

since tdd is the more generic term, that's the one I'd use.